Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+G_gitcred = 'TonJenSSH'
+G_container = "atomxy/empty-ton-sdk-js:20191128"
+G_update = "none"
+G_build = "none"
+G_test = "none"
+
+pipeline {
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '1')
```

Comment:
`numToKeepStr` is set to `1`. Could we keep more than 1 last build? I would propose to use default value, `10`

Reply:
Is I said in another PR I don't think that we need to keep builds for subproject